specification/app/Microsoft.App/routes.tsp

```diff
@@ -21,7 +21,7 @@ alias SessionTraits = NoRepeatableRequests &
 
 alias SessionOperations = Azure.Core.ResourceOperations<SessionTraits>;
 
-interface CodeExecution {
+interface SessionCodeExecution {
   #suppress "@azure-tools/typespec-azure-core/use-standard-operations" ""
   @doc("Execute code in a session.")
   @route("/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/sessionPools/{sessionPoolName}/code/execute")
```

Comment:
`/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/sessionPools/{sessionPoolName}` we can name this as session pool management endpoint as we are not doing anything special for it like returning these information back in response etc.